### ISSUE TITLE
Facebook: Automatically added scope "identify" leads to Error

### ISSUE THREAD

**Issue by Flunsi** [pilcrowonpaper/arctic#4]
The error occurs when redirecting to the authorizationURL.

This causes the error : `this.scope.push("identify");` 

Also "identify" isnt defined in [https://developers.facebook.com/docs/permissions/](https://developers.facebook.com/docs/permissions/)

**Comment by Flunsi** [comment]
Its obviously a copy&paste error, because below that in the getUser()-methode, it tries to get data from discord. 
```
	public async getUser(accessToken: string): Promise<FacebookUser> {
		const url = new URL("https://discord.com/api/users/@me");
```

**Comment by pilcrowonpaper** [comment]
Fixed with `0.3.5`

**Comment by Flunsi** [comment]
It works. thx!
